app/src/main/java/com/sunlightlabs/android/congress/fragments/CommitteeListFragment.java

```diff
@@ -257,7 +258,7 @@ public int compare(Committee a, Committee b) {
 		private List<Committee> forCommittee(String committeeId) {
 			List<Committee> result = new ArrayList<Committee>();
 			try {
-				result = CommitteeService.getSubcommitteesFor(committeeId);
+				result = CommitteeService.find(committeeId).subcommittees;
```

Comment:
Subcommittees are now returned whenever details are found for a parent committee, rather than filtering a flat list of committees and subcommittees.